### DISCUSSION TITLE
E22-900M30S sleep current

### DISCUSSION THREAD

**Discussion by frogg1ven** [jgromes/RadioLib#421] [General]
Hello,
is there any chance to reduce sleep current to 3 uA as in datasheet? Right now when I'm using sleep function, sleep current is about 55 uA.
![image](https://user-images.githubusercontent.com/49206712/145086793-e75fa2cc-a5c9-42cd-9c54-948a208d793b.png)
Regards.

**Comment by jgromes** [comment]
Great question - to be directed at Ebyte, it's their datasheet. Personally, I don't think that value is realistic, cosidering the RF switch and PA Ebyte added. I'd start by asking them what does "Software is shutdown" mean.

**Reply by frogg1ven** [reply]
So do you think that current I managed to achieve (55 uA) is as low as possible right now?

**Reply by jgromes** [reply]
That really depends on the components Ebyte put inside the module. You can try calling `sleep(false)` to not retain config, but even with config retention (`sleep()`), the current draw of the SX126x chip (which is the only part RadioLib can control) should be 0.6 uA according to SX126x datasheet.
